spec/date.html

```diff
@@ -821,7 +821,8 @@ <h1>Properties of Temporal.Date Instances</h1>
       <li>[[ISOYear]], representing the year.</li>
       <li>[[ISOMonth]], representing the month within the year. It lies between 1 and 12, inclusive.</li>
       <li>[[ISODay]], representing the day within the month. It lies between 1 and DaysInMonth([[ISOYear]], [[ISOMonth]]), inclusive.</li>
+      <li>[[Calendar]], representing the calendar system to project the date into.</li>
     </ul>
-    <p>The value of each of these internal slots is a Number which is an integer.</p>
+    <p>The value of each of these internal slots is a Number which is an integer, except for [[Calendar]], whose value is a %Temporal.Calendar% object.</p>
```

Comment:
It can be any object, not just a Temporal.Calendar object, if you have a user calendar represented by an object with the right methods.

Reply:
Done